packages/components/src/DataList/DataList.stories.tsx

```diff
@@ -12,43 +21,113 @@ export default {
     viewMode: "story",
   },
   // Comment this out to make it show up in storybook
-  excludeStories: ["Basic"],
+  // excludeStories: ["Basic"],
   decorators: [
     // Detach from Storybook's layout
-    Story => (
-      <div
-        style={{
-          position: "fixed",
-          inset: 0,
-          overflow: "auto",
-          display: "flex",
-          flexDirection: "column",
-        }}
-      >
-        <Story />
-      </div>
-    ),
+    (Story, { viewMode }) => {
+      if (viewMode === "docs") return <Story />;
+      return (
+        <div
+          style={{
+            position: "fixed",
+            inset: 0,
+            overflow: "auto",
+            display: "flex",
+            flexDirection: "column",
+            padding: "0px 16px 16px",
+          }}
+        >
+          <Story />
+        </div>
+      );
+    },
   ],
 } as ComponentMeta<typeof DataList>;
 
-const Template: ComponentStory<typeof DataList> = args => (
-  <DataList {...args} />
-);
+const Template: ComponentStory<typeof DataList> = args => {
+  const {
+    data,
+    /* See useCollectionQuery for example on how to load more */
+    // refresh,
```

Comment:
When we do the infinite scroll we should show this off (later thing)

Reply:
totally!